### PR TITLE
Add release workflow for github actions

### PR DESCRIPTION
**Pull request by miseyu**
## Summary
- add release workflow for scalardb test and scalardl test docker images

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+      - name: Create scalardl test containers
+        run: ./gradlew docker -PdockerVersion=${{ steps.version.outputs.version }}
+        working-directory: scalardl-test
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.CR_PAT }}
```
Comment (miseyu) [inline .github/workflows/release.yaml:50]: @feeblefakie 
@brfrn169 
Is `CR_PAT` registered in the organization secrets?
If it is a repository-specific secret, please register it in this repository as well. 🙇

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+      - name: Push containers
+        run: |
+          docker push ghcr.io/scalar-labs/kelpie-test-scalardb:${{ steps.version.outputs.version }}
+          docker push ghcr.io/scalar-labs/kelpie-test-scalardl:${{ steps.version.outputs.version }}
+
+      - name: Create release
```
Comment (brfrn169) [inline .github/workflows/release.yaml]: Do we really need to create a release? Maybe only pushing the docker image is enough?

**Review by brfrn169 — COMMENTED** [review]
Thank you! Left a question. Please take a look!

File: .github/workflows/release.yaml (hunk 2 above)
Comment (miseyu) [reply .github/workflows/release.yaml]: @brfrn169 
You're right. I have corrected it.
https://github.com/scalar-labs/kelpie-test/pull/42/commits/f7d3f496d8e496d274210b6e44b0ade83f4d97ea

**Comment by miseyu** [comment]
@brfrn169 
I hope you can confirm that it has been corrected. :bow:
PTAL

**Review by brfrn169 — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**